src/components/UpcomingBirthdays/UpcomingBirthdays.css

```diff
@@ -0,0 +1,20 @@
+.birthdayArea {
+  border: 1px solid #403F3E;
+  float: left;
+  padding: 15px 100px 15px 15px;
+  margin-left: 5px;
+  height: 100vh
```

Comment:
When we add our break points would it make more sense if we used definite things like px/em?

Reply:
Yeah I'm good using definite things. Not super picky on it